app/views/games/_chessboard.html.erb

```diff
@@ -0,0 +1,86 @@
+<div class="container">
+
+
+    <table id="chess-board" class="col-centered" >
+      <tr>
```

Comment:
Looks like a lot of repetition here, what about maybe a series of loops to generate the rows and cols dynamically?